vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -207,22 +220,31 @@ def moe_align_block_size(
 
 
 def invoke_fused_moe_kernel(A: torch.Tensor, B: torch.Tensor, C: torch.Tensor,
-                            topk_weights: torch.Tensor, topk_ids: torch.Tensor,
+                            B_scale: torch.Tensor, topk_weights: torch.Tensor,
+                            topk_ids: torch.Tensor,
                             sorted_token_ids: torch.Tensor,
                             expert_ids: torch.Tensor,
                             num_tokens_post_padded: torch.Tensor,
                             mul_routed_weight: bool, top_k: int,
-                            config: Dict[str, Any]) -> None:
+                            config: Dict[str, Any], compute_type: tl.dtype,
+                            use_fp8: bool) -> None:
     assert topk_weights.stride(1) == 1
     assert sorted_token_ids.stride(0) == 1
 
+    if use_fp8:
+        A, A_scale = ops.scaled_fp8_quant(A)
+    else:
+        A_scale = None
```

Comment:
Worth asserting that B_scale is None as well
```suggestion
        assert B_scale is None
        A_scale = None
```

Reply:
Thanks, I fixed it :)